proc/process.c

```diff
@@ -546,6 +546,7 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	char *snameTab;
 	ptr_t *got;
 	struct _reloc reloc[5];
+	size_t stacksz, gnustacksz = 0;
```

Comment:
`size_t stacksz = SIZE_USTACK`, then we don't need `gnustacksz ` variable

Reply:
Done